### PR TITLE
Add e2e-metal-assisted to TestGrid

### PR DESCRIPTION
**Pull request by YuviGold**

### CODE REVIEW

File: core-services/release-controller/_releases/release-ocp-4.7.json [hunk 1]
```diff
@@ -56,6 +56,9 @@
       "optional":true,
       "prowJob":{"name":"release-openshift-ocp-installer-e2e-metal-4.7"}
     },
+    "metal-assisted":{
```
Comment (stbenjam) [inline core-services/release-controller/_releases/release-ocp-4.7.json:59]: This needs to be `"optional": true` until the job is proven stable

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**